tests/ui/circulation/test_actions_extend.py

```diff
@@ -72,7 +72,8 @@ def test_fees_after_extend(
         checkout_cipo, dbcommit=True, reindex=True)
 
     # UPDATE LOAN TO BE OVERDUE
-    interval = 10
+    # LIBRARY FIXTURES EXCEPTION: Christmas Holidays is 15 days
+    interval = 15
```

Comment:
Does the test also still work if running outside of the exception interval (other date)?

Reply:
Yes I did a test with the date of 2022-06-15 and everything is OK.